### PR TITLE
1906-09-17

### PR DESCRIPTION
**Pull request by icc17**
This is my page one of my Monday issue! I was previously having problems with XML so here it is.

### CODE REVIEW

File: 1906-09-17.xml [hunk 1]
```diff
@@ -0,0 +1,1348 @@
...
+                                <cell>Tabor</cell>
+                                <cell>" 3,000</cell>
+                            </row>
+                        </table>
+                        <p>*Second class accommodation only, unless specially reserved.—Fares : Alexandria
+                            to Liverpool, 1st, £14 Single, £25 Return. 2nd, £9 Single, £15 Return.—To Malta,
+                            1st, £5 Single, £9 Return, 2nd, £3 Single, £5 Return.—Return tickets available
+                            for six months.</p>
+                        <p><name>S.S. Tabor</name> now on the berth, will sail on or about <date
+                            when="1905-09-27">Thursday, September 27</date>th, to be followed by <name>S.S.
```
Comment (whanley) [inline 1906-09-17.xml]: 1906

File: 1906-09-17.xml [hunk 2]
```diff
@@ -0,0 +1,1348 @@
...
+                            <row>
+                                <cell>Friday</cell>
+                                <cell><date when="1906-09-21">21 Sept</date></cell>
+                                <cell>at 4 p.m.</cell>
+                                <cell>Niger</cell>
+                                <cell>Capt. Schmitz</cell>
+                            </row>
+                            <row>
+                                <cell>"</cell>
+                                <cell><date when="1905-07-21">28 "</date></cell>
```
Comment (whanley) [inline 1906-09-17.xml]: 1906

File: 1906-09-17.xml [hunk 3]
```diff
@@ -0,0 +1,1348 @@
...
+                                <cell>Sunday</cell>
+                                <cell><date when="1906-09-10">9 "</date></cell>
+                                <cell>Natal</cell>
+                                <cell>" Girard</cell>
+                                <cell>" " China</cell>
+                            </row>
+                            <row>
+                                <cell>" "</cell>
+                                <cell>Thursday</cell>
+                                <cell><date when="1905-09-27">27 "</date></cell>
```
Comment (whanley) [inline 1906-09-17.xml]: 1906

File: 1906-09-17.xml [hunk 4]
```diff
@@ -0,0 +1,1348 @@
...
+                                <cell>Thursday</cell>
+                                <cell><date when="1906-09-20">20 "</date></cell>
+                                <cell>Diemnah</cell>
+                                <cell>" Durrande</cell>
+                                <cell>" " China</cell>
+                            </row>
+                            <row>
+                                <cell>" "</cell>
+                                <cell>Sunday</cell>
+                                <cell><date when="1905-09-16">16 "</date></cell>
```
Comment (whanley) [inline 1906-09-17.xml]: 1906

File: 1906-09-17.xml [hunk 5]
```diff
@@ -0,0 +1,1348 @@
...
+                        <p> F. G. DAVIDSON, Superintendent P. &amp; O. S. N. Company in Egypt SUEZ.</p>
+                    </div>
+                    <div type="advert" xml:id="deg-ad-orm01">
+                        <head>Orient-Royal Mail Line.</head>
+                        <p>OUTWARD to AUSTRALIA.</p>
+                        <p><name>R.M.S. “Oruba"</name> will leave Suez about <date when="1906-10-06">October 6</date></p>
+                        <p><name>R.M.S "Orontes"</name> will leave Suez about <date when="1906-09-21">September 21</date>.</p>
+                        <p>HOMEWARD to NAPLES, MARSEILLES, GIBRALTAR, PLYMOUTH, LONDON, TILBURY.</p>
+                        <p><name>R.M.S. "Omrah"</name> will leave Port Said <date when="1906-09-25">September 25</date></p>
+                        <p><name>R.M.S. “Oroya"</name> will leave Port Said <date when="1905-10-08">October 8</date></p>
```
Comment (whanley) [inline 1906-09-17.xml]: 1906

File: 1906-09-17.xml [hunk 6]
```diff
@@ -0,0 +1,1348 @@
...
+                            Trieste-Vienna-Ostende. Passengers reach London every Friday 4.50 p.m.</p>
+                        <table rows="3" cols="8">
+                            <row>
+                                <cell><date when="1906-08-04">Aug. 4</date></cell>
+                                <cell>4 p.m.</cell>
+                                <cell><name>S.S. Semiramis</name></cell>
+                                <cell><persName>Capt. Martinolich</persName></cell>
+                                <cell><date when="1906-09-01">Sept. 1</date></cell>
+                                <cell>4 p.m.</cell>
+
```
Comment (whanley) [inline 1906-09-17.xml]: these column markers should be distributed within the page

**Comment by icc17** [comment]
I fixed the problems with page 1 and also worked on most of page 2!

**Comment by whanley** [comment]
Deliverable 1 is complete. Good! Page 2 is well started. Is your OCR working? If not, straightening your images will probably help. Don't hesitate to ask if Weber or I can give you a hand.

**Comment by whanley** [comment]
Course ended without completion.

**Pull request closed without merging**